docgen/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "docgen"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.83"
+serde = { version = "1.0.201", features = ["derive"] }
+serde_yaml = "0.9.34"
+tokio = "1.37.0"
+zqutils = { git = "https://github.com/zilliqa/zq-base", branch = "users/richard/20" }
```

Comment:
Are you sure we want to depend on a non-`main` branch? It might lead to pain later on.

Reply:
No - sorry! https://github.com/Zilliqa/zq-base/pull/23 ...